src/NLog.StructuredLogging.Json.Tests/LoggingException.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace NLog.StructuredLogging.Json.Tests
+{
+    [Serializable]
```

Comment:
Ditto on this (just one [example here](https://github.com/nunit/nunit/blob/8a9195879a7bd7f23231e5584e54663305c19597/src/NUnitFramework/framework/Exceptions/AssertionException.cs#L33))
I think maybe it's redundant due to binary serialization being not present yet (coming in netstandard2.0).